README.md

```diff
@@ -52,3 +53,51 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+
+## Testing the App
+
+Once all the dependancies have been installed, start testing by running:
+```bash
+$ pytest
```

Comment:
This will work if you've manually activated your virtual environment, or if your global environment happens to be correctly configured, but generally safer to run through poetry (`poetry run pytest`) just to ensure that you are definitely using the virtual environment